SwiftPaginator/SwiftPaginator.swift

```diff
@@ -114,6 +124,15 @@ public class Paginator<Element> {
     return page >= Int(totalPages)
   }
   
+  /// Have all results been fetched
+  private var fetchedAllResults: Bool {
+    if results.count == total {
```

Comment:
I could...i feel there should never be a case when the results are greater than total, but I'll double check to see how much of a possibility that is